web/src/pages/services.tsx

```diff
@@ -4,6 +4,45 @@ import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 import { ServiceList } from '../components/services-page/service-list';
 
+const boxesData = {
+  dividedImage: 'ServicesSplit.png',
```

Comment:
This would help with internalization as well, as we can just put markdown files in different folders (e.g. `web/src/en/services/...` and `web/src/nb/services/...`).

If we agree on this approach, we can almost have a rule like this: no data amongst visual components.

Feedback?

Reply:
👆 I forgot about #106 🙄

Adding all these comments over there :)